test/test.go

```diff
@@ -505,6 +505,10 @@ func (h *Helper) Path(name string) string {
 		joined = filepath.Join(h.tempdir, name)
 	}
 
+	if _, err := os.Stat(joined); os.IsNotExist(err) {
+		return ""
+	}
+
 	// Ensure it's the absolute, symlink-less path we're returning
 	abs, err := filepath.EvalSymlinks(joined)
```

Comment:
Oh, I see - ProjPath (which calls TestHelper.Path) is getting called.  I'd be comfortable rewriting ProjPath to be just what I had originally intended: $TMPDIR + ROOT + path, rather than running the symlink check.  @sdboyer, does the symlink check called by TestHelper.Path really need to be done for these temp projects where all contents were copied over from testdata?